scripts/cluster_launcher.py

```diff
@@ -98,7 +98,7 @@ def parseJobsFile(self, template_dir, job_file):
         return jobs
 
     def createAndLaunchJob(self, template_dir, job_file, specs, options):
-        next_dir = getNextDirName(specs['job_name'], os.listdir('.'))
+        next_dir = getNextDirName(specs['job_name'], os.listdir(template_dir))
```

Comment:
I am finding issue with this change... I didn't realize I was updating the issue instead of the PR:

https://github.com/idaholab/moose/issues/8750